lib/Client.js

```diff
@@ -1,32 +1,18 @@
 "use strict";
 
-const ApplicationCommand = require("./structures/ApplicationCommand");
 const Base = require("./structures/Base");
-const Channel = require("./structures/Channel");
 const Collection = require("./util/Collection");
 const Constants = require("./Constants");
 const Endpoints = require("./rest/Endpoints");
-const ExtendedUser = require("./structures/ExtendedUser");
 const Guild = require("./structures/Guild");
-const GuildAuditLogEntry = require("./structures/GuildAuditLogEntry");
-const GuildIntegration = require("./structures/GuildIntegration");
-const GuildPreview = require("./structures/GuildPreview");
-const GuildTemplate = require("./structures/GuildTemplate");
-const GuildScheduledEvent = require("./structures/GuildScheduledEvent");
-const Invite = require("./structures/Invite");
-const Member = require("./structures/Member");
-const Message = require("./structures/Message");
-const Permission = require("./structures/Permission");
 const PrivateChannel = require("./structures/PrivateChannel");
-const RequestHandler = require("./rest/RequestHandler");
-const Role = require("./structures/Role");
+const RESTClient = require("./rest/RESTClient");
 const ShardManager = require("./gateway/ShardManager");
-const StageInstance = require("./structures/StageInstance");
-const ThreadMember = require("./structures/ThreadMember");
 const UnavailableGuild = require("./structures/UnavailableGuild");
 const User = require("./structures/User");
 const VoiceConnectionManager = require("./voice/VoiceConnectionManager");
-const AutoModerationRule = require("./structures/AutoModerationRule");
+const {formatAllowedMentions} = require("./util/util");
+const {setTimeout: sleep} = require("node:timers/promises");
```

Comment:
I'd be more in favor of keeping the universal JS variant for promisified sleep here

```suggestion
const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
```